### DISCUSSION TITLE
slow regulation / calibration offset

### DISCUSSION THREAD

**Discussion by maia** [jmcollin78/versatile_thermostat#203] [General]
Here is an example for the calibration offset in my bathroom, which is working okay but not strong enough:

1. The target temperature is 22°C.
2. The internal TRV sensor thinks the room temperature is about 24°C. 
3. As this is above the regulated temperature of 23°, the valve stays rather closed.
4. The room temperature stays at 21,5°C, as it's not heated strongly.

<img width="435" alt="Bildschirmfoto 2023-11-18 um 12 48 14" src="https://github.com/jmcollin78/versatile_thermostat/assets/9836/fd3ce2f5-607e-4bca-9324-2810d5b0350e">

This shows that the "slow" regulation could be a bit stronger over the long term.

**Comment by jmcollin78** [comment]
You have now with the 4.1.0 the possibility to specify your own parameters rather than using the pre-configured one.
That is a good use case for that. See https://github.com/jmcollin78/versatile_thermostat#self-regulation-in-expert-mode

(and you are an expert)
